packages/bot-web-ui/src/stores/quick-strategy-store.js

```diff
@@ -66,17 +76,25 @@ export default class QuickStrategyStore {
 
     @action.bound
     setSelectedDurationUnit(duration_unit) {
-        this.selected_duration_unit = duration_unit;
+        this.qs_cache.selected_duration_unit = this.selected_duration_unit = duration_unit;
```

Comment:
Please don't use multi-assign for clarity, it's completely fine to assign the same value to different vars on multiple lines. Please see https://eslint.org/docs/rules/no-multi-assign